src/discordcr/rest.cr

```diff
@@ -620,6 +620,59 @@ module Discord
       Guild.from_json(response.body)
     end
 
+    # Gets a list of emojis. Will not work for bots (it'll return an empty array).
```

Comment:
You should clarify these endpoints only work for whitelisted OAuth2 apps.

Reply:
Since this PR opened, managing emoji is now a public API :tada: 

You can adjust the docstrings to reflect this (it *will* work for bots). Now however, it is only the `roles` field of an emoji that requires whitelisting to create/modify.

Reply:
Don't have time to do this right now, if somebody does then go ahead and I'll close this PR